Docs/Product/Circuit_Breaker_Unification_PRD.md

```
@@ -97,28 +101,49 @@ Circuit Breaker Unification PRD
       - Decorator factory circuit_breaker(...) (sync/async support).
   - Configuration resolution:
       - Accept explicit config from call site; otherwise resolve via per-category sources:
-          - Embeddings: tldw_Server_API/Config_Files/embeddings_production_config.yaml (circuit_breaker block)
+          - Embeddings: tldw_Server_API/Config_Files/embeddings_production_config.yaml (retry.circuit_breaker block)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 4; Actual: 10

(MD007, ul-indent)